libs/pipelines/api_v2_pipeline.py

```diff
@@ -35,7 +34,7 @@
 class RegionalInput:
     region: pipeline.Region
 
-    _combined_data: combined_datasets.RegionalData
+    _combined_data_with_test_positivity: OneRegionTimeseriesDataset
```

Comment:
I'm wondering about the naming here.  we pass rt_data + icu_data as separate one region datasets and combined data.  Also, technically the data is still "combined data" if it is normal data combined with test positivity calculation.

This is probably less of a comment for this PR and more of a comment about some sort of schema for the dataset classes.  I've been thinking for a bit if there was a way to surface the columns that are available better.

Reply:
Agreed the name isn't great, but feels more explicit than a comment saying the same thing. I'm also not super keen on the way that rt_data and icu_data are handled differently from test positivity. Some way to declare the columns in a way that plays well with python would be nice but hasn't gotten swept up in any of my restructuring yet.